scripts/windows_install_vs.ps1

```diff
@@ -30,6 +30,7 @@ function InstallVS($Version, $Link)
 
 InstallVS "2019" "https://aka.ms/vs/16/release/vs_buildtools.exe"
 InstallVS "2022" "https://aka.ms/vs/17/release/vs_buildtools.exe"
+InstallVS "2026" "https://aka.ms/vs/stable/vs_buildtools.exe"
```

Comment:
Won't that mean we'll automatically use a newer VS version when it's released, since the new URL isn't versioned? Is there a versioned URL?

Reply:
You are right but unfortunately there is no versioned URL for VS 2026. Please check https://developercommunity.visualstudio.com/t/Missing-Version-Specific-Download-Link-f/11002499.

Reply:
> I think we at least need to add a warning comment about this.

Done.